tests/model_fields/tests.py

```diff
@@ -278,19 +286,19 @@ def test_blank_in_grouped_choices(self):
             ("b", "Bar"),
             (
                 "Group",
-                (
+                [
                     ("", "No Preference"),
                     ("fg", "Foo"),
                     ("bg", "Bar"),
-                ),
+                ],
```

Comment:
This looks unrelated. Please revert.

Reply:
It was putting the choices in the normal list of 2-tuples form, but I guess it doesn't matter as we can handle it. Reverting...

Reply:
Oops. Reverted this one back again too. It's required as we properly normalize nested choices too.